themes/startwords/assets/js/notes.js

```diff
@@ -0,0 +1,58 @@
+/* javascript for notes behavior */
+
+window.onload = function() {
+	// position contextual note if the page is loaded with one selected
+	// FIXME: not displaying on load - is this due to scrolling ?
+	positionContextualNote();
+};
+
+window.addEventListener('hashchange', function() {
+	positionContextualNote();
```

Comment:
this is an interesting hook i didn't know about. we might need to make sure we don't load this js on pages other than the article page, because some pages (like the issue detail page) use anchor links to move the user around (like when you click the chevron) but have no relationship to contextual notes. although I guess the function could fire and do nothing...

Reply:
I had a question about whether contextual notes should work on any pages or only on articles - in theory we could have markdown notes on other content pages, but IDK if we'd want them to behave like this or not.

The function looks for fn style hashes and should ignore everything else.

Reply:
good point - should be safe to leave it in, then.